### PR TITLE
Spelling and grammar fixes *FIXED*

### PR DESCRIPTION
**Pull request by pxlsr**

### CODE REVIEW

**Comment by SamuelRowe** [comment]
Sweet :+1:

File: code/modules/food&drinks/drinks/drinks/drinkingglass.dm [hunk 1]
```diff
@@ -470,7 +470,7 @@
 			if("gibbfloats")
 				icon_state = "gibbfloats"
 				name = "Gibbfloat"
-				desc = "Dr. Gibb with ice cream on top."
```
Comment (Dorsisdwarf) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: This isn't actually a grammar error. Ice Cream is actually the correct way to write it.

File: code/modules/food&drinks/drinks/drinks/drinkingglass.dm [hunk 2]
```diff
@@ -158,7 +158,7 @@
 			if("bravebull")
 				icon_state = "bravebullglass"
 				name = "Brave Bull"
-				desc = "tequila and Coffee liquor, brought together in a mouthwatering mixture. Drink up."
+				desc = "Tequila and Coffee liqueur, brought together in a mouthwatering mixture. Drink up."
```
Comment (Dorsisdwarf) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: Except that the drink in SS13 is "RR Coffee Liqour", not coffee liqueur.

Comment (SamuelRowe) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: I'll make the appropriate changes. Thanks!

File: code/modules/food&drinks/drinks/drinks/drinkingglass.dm [hunk 3]
```diff
@@ -450,7 +450,7 @@
 			if("vanillapudding")
 				icon_state = "vanillapudding"
 				name = "Vanilla Pudding"
-				desc = "Whoah, this thing is on FIRE"
+				desc = "Whoah, this thing is on FIRE!"
```
Comment (Cheridan) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: Sir, my pudding is still on fire! (this is a copy-paste from toxin special)

**Pull request merged**